.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,4 @@
 ### Information
-- Fixes #_.
```

Comment:
Why are we removing this?

Reply:
I would think that "Fixes" is somewhat redundant to "JIRA story".  I usually just put the JIRA number here and the actual changes in the next section.

Reply:
^ what Scott said

Reply:
I could see this as a repeat of the Jira ticket's title or description. I usually just delete the Fixes and add the ticket number. I would be further inclined to just change it to say something like `Fixes or JIRA Ticket`. The reason why Fixes is there is if there are any developers who aren't on our JIRA working on fixes for this project. But I like it better in one line.

Reply:
This line was added originally for the users who create a PR that fixes a GitHub issue.  GitHub keys off `Fixes #23` which will close GitHub issue 23 as fixed once the PR is merged into `master`.